src/components/member/tabs/contract-tab/norwegian-travel-agreement-key-info.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { Card } from '../../../../../shared/hedvig-ui/card'
+import { Contract, NorwegianTravel } from '../../../../api/generated/graphql'
+import { FlexWrapper } from './contract-item'
+
+export const NorwegianTravelAgreementKeyInfoComponent: React.FunctionComponent<{
+  contract: Contract
+  agreement: NorwegianTravel
+}> = ({ contract, agreement }) => {
+  return (
+    <FlexWrapper>
+      <Card span={2}>
+        <span>name: </span>
+        <span>Type: {agreement.__typename}</span>
+        <span>Number of insureds: {agreement.numberCoInsured + 1}</span>
```

Comment:
So far you've used, household size, number of people and number of insureds 😄 
```suggestion
        <span>Number co-insured: {agreement.numberCoInsured}</span>
```